src/meta/src/hummock/hummock_manager.rs

```diff
@@ -261,18 +261,19 @@ where
                     .levels
                     .insert(compaction_group.group_id(), Levels { levels });
             }
-            if version_ids.is_empty() {
-                version_ids.insert(init_version.id);
-            }
-            let mut redo_state = init_version.clone();
-            if version_ids.contains(&init_version.id) {
-                init_version.insert(self.env.meta_store()).await?;
-                versioning_guard
-                    .hummock_versions
-                    .insert(init_version.id, init_version);
-            }
+            version_ids.insert(init_version.id);
+            init_version.insert(self.env.meta_store()).await?;
+        }
+
+        let mut redo_state = HummockVersion::select(self.env.meta_store(), &0)
```

Comment:
`select(...., &0)` looks strange. Can we reuse the result of `HummockVersion::list` in L223 here?

Reply:
> `select(...., &0)` looks strange. Can we reuse the result of `HummockVersion::list` in L223 here?

wallace wants to use a fixed key for this singleton in reviews above

Reply:
I mean we already get all HummockVersion in the returned value of `HummockVersion::list` so we don't need to call `select` to get the first HummockVersion.

Reply:
> I mean we already get all HummockVersion in the returned value of `HummockVersion::list` so we don't need to call `select` to get the first HummockVersion.

fixed